### DISCUSSION TITLE
Finandy denied access from {IP}

### DISCUSSION THREAD

**Discussion by rifkidocs** [soranoo/TradingView-Free-Webhook-Alerts#20] [Q&A] [answered]
Send webhook failed.
Reason: {"success":false,"error":"Hook 111501 access denied from 44.192.112.189"}
Status: 400

Domain: hook.finandy.com
Full URL: https://hook.finandy.com/yNhGy93lJ4SS1Sfmq1UK
Content: {'name': 'Racik', 'secret': '46qlo*****', 'side': 'buy', 'symbol': 'ETHUSDTPERP', 'positionSide': 'flat', 'open': {'amountType': 'amount', 'amount': '0.611', 'leverage': '10'}}

how to fix this?

**Comment by soranoo** [comment, marked as answer]
Had you entered `Allowed IP`? Try to enter your host machine's IP `44.192.112.189`. Or you can find out your IP at https://myip.com.

Don't forget to click the save button. 💯

**Comment by soranoo** [comment]
Just like this.
![image](https://user-images.githubusercontent.com/46896789/209204600-1a87562b-8d1e-41ac-9ae4-a263c946bb7e.png)

**Reply by rifkidocs** [reply]
OK
Webhook Sent!

Process Time: 527.307ms
Domain: hook.finandy.com
Full URL: https://hook.finandy.com/yNhGy93lJ4SS1Sfmq1UK
Content: {'name': 'Racik', 'secret': '46qlok*****', 'side': 'buy', 'symbol': 'ETHUSDTPERP', 'positionSide': 'long', 'open': {'amountType': 'amount', 'amount': '0.367', 'leverage': '10'}}

Developed by Freeman(soranoo)

Thanks it's working now, i put `0.0.0.0/0` on the `Allowed IP` 
![Screenshot_49](https://user-images.githubusercontent.com/105924697/209233455-c3f0bc22-25e0-4944-ace3-17e74a1f67b6.png)
